joomla/joomla-cms
# [4] Change update cache timeout to 12h by default

### Summary of Changes

Proposing to increase the update cache timeout from 6 hours to 12 hours. I hope I found all the places this is needed.

Sometime in history, someone decided 6 was a sensible default. 

**As shown in the last week, getting an email every 6 hours about an available Joomla update is forcing people, [even seasoned Joomla people](https://twitter.com/jen4web/status/1429884632154189824), to disable the Update Notification Emails - not really what we want people to do.** 

Further evidence that the value of these notification is diminished by their frequency can be seen [by the number of articles and videos telling people how to disable the emails on their sites ](https://www.google.com/search?q=disable+joomla+update+notifications)and ignore the fact that updates are available. 

There is no need to check for updates every 6 hours - for three of those checks every 24 hours I'm probably asleep. 

This PR proposes to move to 12 hour timeouts for the cache - meaning that: 

 - Joomla potentially will check less for updates
 - Joomla will email update notification emails only once during any persons waking day... I consider a single email a day, and one for me to wake up to, better than one every 6 hours, so when I wake up I get 2 or 3 duplicated emails in my mailbox. 

This PR and the proposal is NOT to change existing sites timeout settings. Therefore there is no data migration planned in this PR, but to have this new setting only going forward (also why no Joomla 3 version of this PR) 

### Testing Instructions

Check for updates. Wait 12 hours, visit your site, see if you get update notification emails. 

### Actual result BEFORE applying this Pull Request

potentially 4 emails per day

### Expected result AFTER applying this Pull Request

potentially 2 emails per day
Update sites under less stress/strain
Cheaper CDN payments for extension developers :) 

### Documentation Changes Required

Probably. 

### Conflicts

Will eventually conflict with typo PR in #35346
